### DISCUSSION TITLE
Aura Light?

### DISCUSSION THREAD

**Discussion by jvzaniolo** [daltonmenezes/aura-theme#33] [Ideas]
It would be nice to have a light variant with same colorscheme

**Comment by daltonmenezes** [comment]
Hi @jvzaniolo , there's no official plans to support light variants, the main Aura's tagline is "dark" and if we make a lot of variants is complicated to support and host in a lot of stores that can't support variants in the same package, and we have to host one by one separately, this makes the work painful to maintain as Aura grows, but you are free to fork and make it by your own as a side project without problems. 💜
